### PR TITLE
Log node host name resolution exception

### PR DESCRIPTION
**Pull request by kokosing**
Log node host name resolution exception

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java [hunk 1]
```diff
@@ -156,6 +159,7 @@ public NodeSelector createNodeSelector(ConnectorId connectorId)
                 }
                 catch (UnknownHostException e) {
                     // ignore
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java]: remove

File: presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java [hunk 2]
```diff
@@ -156,6 +159,7 @@ public NodeSelector createNodeSelector(ConnectorId connectorId)
...
+                    LOG.debug(e, "Unable to resolve host name for: " + node);
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java]: `.error`? `.warn`?

"for:" -> "for node:"

Comment (electrum) [reply presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java]: Use strong formatting. This constructs the string even if the log level is disabled.

Comment (electrum) [inline presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java]:
```suggestion
                    LOG.debug(e, "Unable to resolve host name for node: %s", node);
```

**Comment by dain** [comment]
@cla-bot check

**Comment by kokosing** [comment]
Comments addressed.

**Pull request merged**